p2p/src/sync/mod.rs

```diff
@@ -72,6 +76,16 @@ impl<T> FatalError for error::Result<T> {
     }
 }
 
+enum RequestType {
+    GetHeaders,
+    GetBlocks(Id<Block>),
```

Comment:
Should it be `GetBlock`? It only takes one block ID.

Reply:
I'll rename it. It's called `GetBlocks` because we have a message called `GetBlocks` which takes a vector (right now the vector can only contain one element). It's a mess

Reply:
Actually this should be a `Vec<Id<Block>>`. But it was set to this temporarily, IIUC.

Reply:
I'll use a vector

Reply:
Don't forget to enforce a limit on the size of this vector.